docs/sources/explore/trace-integration.md

```diff
@@ -8,167 +7,159 @@ labels:
     - cloud
     - enterprise
     - oss
-title: Tracing in Explore
+title: Traces in Explore
 weight: 20
 ---
 
-# Tracing in Explore
+# Traces in Explore
 
-You can use Explore to query and visualize traces from tracing data sources.
+You can use Explore to query and visualize traces from tracing data sources. Supported data sources include:
 
-Supported data sources are:
-
-- [Tempo]({{< relref "../datasources/tempo/" >}}) (supported ingestion formats: OpenTelemetry, Jaeger, and Zipkin)
-- [Jaeger]({{< relref "../datasources/jaeger/" >}})
-- [Zipkin]({{< relref "../datasources/zipkin/" >}})
+- [Tempo](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/)
+- [Jaeger](/docs/grafana/<GRAFANA_VERSION>/datasources/jaeger/)
+- [Zipkin](/docs/grafana/<GRAFANA_VERSION>/datasources/zipkin/)
 - [X-Ray](https://grafana.com/grafana/plugins/grafana-x-ray-datasource)
-- [Azure Monitor Application Insights]({{< relref "../datasources/azure-monitor/" >}})
+- [Azure Monitor](/docs/grafana/latest/datasources/azure-monitor/)
 - [ClickHouse](https://github.com/grafana/clickhouse-datasource)
-- [New Relic](https://grafana.com/grafana/plugins/grafana-newrelic-datasource)
-- [Infinity](https://grafana.com/grafana/plugins/yesoreyeram-infinity-datasource)
+- [New Relic](/docs/plugins/grafana-newrelic-datasource/latest/)
+- [Infinity](/docs/plugins/yesoreyeram-infinity-datasource/latest/)
 
-For information on how to configure queries for the data sources listed above, refer to the documentation for specific data source.
+If you are getting started working with traces in Tempo refer to the following documents:
```

Comment:
This is a tricky one. Currently, there isn't a UI for Tempo. It's a database for storing tracing data. How is this rewording? Also, is it going to be confusing for people to see Get started with Tempo in a document talking about Traces in Explore? 

```suggestion
Here are some references to learn more about traces and how you can use them:  
```